spec/features/user_behaviours_spec.rb

```diff
@@ -53,7 +53,7 @@
       sign_out
       visit city_path(@user.home_city)
       click_link('5 spots left')
-      fill_in :name, with:  @user.name
+      fill_in 'tea_time_nickname', with:  @user.name
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.